BUILD.md

```diff
@@ -37,9 +37,9 @@ To build a container image of `openshift-ansible` using standalone **Docker**:
 
 To build an openshift-ansible image using an **OpenShift** [build and image stream](https://docs.openshift.org/latest/architecture/core_concepts/builds_and_image_streams.html) the straightforward command would be:
 
-        oc new-build registry.centos.org/openshift/playbook2image~https://github.com/openshift/openshift-ansible
+        oc new-build openshift/origin-ansible~https://github.com/openshift/openshift-ansible
```

Comment:
that doesn't look right... `openshift/origin-ansible` is not a builder image...

I think it's not that easy to build the image in OpenShift anymore... the Dockerfile is under images/ but the context is the tld... 

Maybe we can just leave out the whole *Building on OpenShift* section for now?